easybuild/easyconfigs/f/flex/flex-2.6.4.eb

```diff
@@ -12,6 +12,8 @@ description = """
 toolchain = {'name': 'dummy', 'version': ''}
 toolchainopts = {'pic': True}
 
+buildopts = 'CFLAGS="$CFLAGS -D_GNU_SOURCE"'
```

Comment:
Even better seems to be:
preconfigopts = 'export CPPFLAGS="$CPPFLAGS -D_GNU_SOURCE" && '

Reply:
With CPPFLAGS instead of CFLAGS the -O2 that configure adds doesn't get lost.